main_test.go

```diff
@@ -76,13 +86,13 @@ func TestInvalidXML(t *testing.T) {
 
 	g.Expect(err).NotTo(BeNil())
 	g.Expect(status.ResponseType).To(Equal("Error"))
-	g.Expect(status.Response).To(Equal("corrupted response"))
+	g.Expect(status.Response).To(Equal("request failed"))
 }
 
 func TestStatusCodeNotZero(t *testing.T) {
 	g := NewGomegaWithT(t)
 	_, status, err := client.Request("/status/code/1")
 
 	g.Expect(err).NotTo(BeNil())
-	g.Expect(status.ResponseTypeNumeric).To(Equal(1))
+	g.Expect(status.ResponseTypeNumeric).To(Equal(0))
```

Comment:
I don't understand the change here. Moreover, the function is called `TestStatusCodeNotZero`, which is the opposite of what is done.